src/vfs.c

```diff
@@ -1140,6 +1077,9 @@ static int vfsWalFileRead(sqlite3_file* file, void* buf, int amount, sqlite3_int
 static int vfsWalFileWrite(sqlite3_file* file, const void* buf, int amount, sqlite3_int64 offset)
 {
 	struct vfsWalFile *f = (struct vfsWalFile *)file;
+	/* == Safety ==
+	 * See vfsWalFile read.
```

Comment:
```suggestion
	 * See [vfsWalFileRead].
```
In general I am not sure about whether it's better to use `[identifier]` everywhere or not.

Reply:
For now, let's just use normal names.